torch/utils/_sympy/functions.py

```diff
@@ -101,9 +102,9 @@ def _is_symbols_binary_summation(expr: sympy.Expr) -> bool:
     )
 
 
-def _keep_float(f: Callable[..., _T]) -> Callable[..., Union[_T, sympy.Float]]:
+def _keep_float(f: Callable[[*_Ts], _T]) -> Callable[[*_Ts], Union[_T, sympy.Float]]:
```

Comment:
'*' isn't supported on legacy Python versions, use typing_extensions.Unpack

Reply:
Ah right, ty, will use Unpack! Also switched to using Optional in some other signatures for backwards compatibility.